docs/README.md

```diff
@@ -74,6 +74,20 @@ or the shortcut
 cz c
 ```
 
+### Integrating with Pre-commit
+Commitizen can lint your commit message for you with `cz check`.
+You can integrate this in your [pre-commit](https://pre-commit.com/) config with:
+```yaml
+---
+repos:
+  - repo: https://github.com/commitizen-tools/commitizen
+    rev: v1.17.0
```

Comment:
I would suggest updating the version to the latest commitizen

Reply:
Wouldn't it be better if we just leave it at `master`. That's actually what I use for our company projects.

Reply:
My reason for not doing so is that pre-commit caches repos. You might not notice that you're still using the older version.

Reply:
True, but it's the same with using a fixed version. Also, this is the README. We'd have to update the version every time we bump.

Reply:
I think we'll be documenting `pre-commit` for them with that. It's already documented by pre-commit themselves. But if you guys want I'll put it in. Just let me know.